### DISCUSSION TITLE
Version 3.17.0 released, but not posted on GitHub Releases?

### DISCUSSION THREAD

**Discussion by JamesDBartlett3** [TabularEditor/TabularEditor3#1366] [🙋♀️TE3 Q&A] [answered]
Anyone know when [GitHub Releases](https://github.com/TabularEditor/TabularEditor3/releases) will be updated with the latest version of TE3?

**Comment by otykier** [comment, marked as answer]
That's an oversight, sorry! (we're still working on fully automating our release process). It's here now: https://github.com/TabularEditor/TabularEditor3/releases/tag/3.17.0
